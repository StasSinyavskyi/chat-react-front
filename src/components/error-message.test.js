/* eslint-env jest */
import renderer from 'react-test-renderer';
import React from 'react';
import ReactDOM from 'react-dom';
import ErrorMessage from './error-message';


// if we have several test we can join them in describe
describe('<ErrorMessage />', () => {
  const err = new Error('Test error');
  const testParam = (
    <ErrorMessage
      clearErrors={() => 'clearErrors'}
      errors={err}
    />
  );

  it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(testParam, div);
    ReactDOM.unmountComponentAtNode(div);
  });

  it('render correctly', () => {
    const tree = renderer
      .create(testParam)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
