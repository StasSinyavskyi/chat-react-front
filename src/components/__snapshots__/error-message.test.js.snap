// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ErrorMessage /> render correctly 1`] = `null`;
